src/pint/event_toas.py

```diff
@@ -581,10 +585,12 @@ def load_RXTE_TOAs(eventname, minmjd=-np.inf, maxmjd=np.inf):
     --------
     :func:`get_RXTE_TOAs`
     """
-    return load_event_TOAs(eventname, "xte", minmjd=minmjd, maxmjd=maxmjd)
+    return load_event_TOAs(
+        eventname, "xte", minmjd=minmjd, maxmjd=maxmjd, errors=errors
+    )
 
 
-def load_NICER_TOAs(eventname, minmjd=-np.inf, maxmjd=np.inf):
+def load_NICER_TOAs(eventname, minmjd=-np.inf, maxmjd=np.inf, errors=1 * u.us):
```

Comment:
Please make this `0.1*u.us`

Reply:
Do you want that to be across all event files?  Again, I don't know that we should put too much physical meaning here.

Reply:
Only NICER event files have 100 ns absolute accuracy to UTC.  The other missions are all worse